src/pretix/base/models/items.py

```diff
@@ -369,10 +369,40 @@ def check_quotas(self, ignored_quotas=None, count_waitinglist=True, subevent=Non
         return min([q.availability(count_waitinglist=count_waitinglist, _cache=_cache) for q in check_quotas],
                    key=lambda s: (s[0], s[1] if s[1] is not None else sys.maxsize))
 
+    def allow_delete(self):
+        from pretix.base.models.orders import CartPosition, OrderPosition
+
+        return (
+            not OrderPosition.objects.filter(item=self).exists()
+            and not CartPosition.objects.filter(item=self).exists()
+        )
+
     @cached_property
     def has_variations(self):
         return self.variations.exists()
 
+    @staticmethod
+    def clean_per_order(min_per_order, max_per_order):
+        if min_per_order is not None and max_per_order is not None:
+            if min_per_order > max_per_order:
+                raise ValidationError(_('max_per_order must be greater than min_per_order (or null for no limitation).'))
```

Comment:
In case we also want to use this for the frontend, let's rephrase it more human-friendly (that is also understood by a user who does not know the field names) as e.g. ``The maximum number per order can not be lower than the minimum number per order``.